client/my-sites/checkout/cart/test/cart-item-unmocked.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { CartItem } from '../cart-item';
+import {
+	PLAN_BUSINESS_2_YEARS,
+	PLAN_JETPACK_PERSONAL,
+	PLAN_PERSONAL,
+	PLAN_BLOGGER,
+	PLAN_PREMIUM,
+} from '@automattic/calypso-products';
+
+jest.mock( '@automattic/calypso-config', () => {
+	const fn = () => {};
+	fn.isEnabled = jest.fn( () => null );
+	return fn;
+} );
+jest.mock( '@automattic/format-currency', () => ( {
+	getCurrencyObject: ( price ) => ( { integer: price } ),
+} ) );
+jest.mock( 'calypso/lib/products-values', () => ( {
+	isPlan: jest.fn( () => null ),
+	isTheme: jest.fn( () => null ),
+	isMonthly: jest.fn( () => null ),
+	isYearly: jest.fn( () => null ),
+	isBiennially: jest.fn( () => null ),
+	isBundled: jest.fn( () => null ),
+	isDomainProduct: jest.fn( () => null ),
+	isCredits: jest.fn( () => null ),
+	isGSuiteOrExtraLicenseOrGoogleWorkspace: jest.fn( () => null ),
+} ) );
+
+const cartItem = {
+	cost: 120,
+	bill_period: 1,
+	volume: 1,
+	product_name: 'name!',
+	product_slug: 'plan_value_bundle',
+};
+
+const translate = jest.fn( identity );
+const props = {
+	cartItem,
+	translate,
+};
+
+describe( 'cart-item', () => {
+	describe( 'calcMonthlyBillingDetails - real callbacks', () => {
+		const expectations = [
```

Comment:
Jest provides a pattern for this case, `test.each()` (https://jestjs.io/docs/api#testeachtablename-fn-timeout).

Otherwise eslint will complain that we are using a composed test name in line 82.

Reply:
Thanks. I just moved these tests into their own file to resolve mocking issues (these don't use mocking but the others surrounding their old location did). However, since I'm creating this file I might as well try to clean this up a bit.